cases/0012/0012.py

```diff
@@ -0,0 +1,68 @@
+from petabtests import *
+from petab.C import *
+import petab
+
+import pandas as pd
+
+
+test_id = 12
+
+# problem --------------------------------------------------------------------
+
+model = DEFAULT_MODEL_FILE
+
+condition_df = pd.DataFrame(data={
+    CONDITION_ID: ['c0'],
+    'compartment': [3],
+}).set_index([CONDITION_ID])
+
+measurement_df = pd.DataFrame(data={
+    OBSERVABLE_ID: ['obs_a', 'obs_a'],
+    SIMULATION_CONDITION_ID: ['c0', 'c0'],
+    TIME: [0, 10],
+    MEASUREMENT: [0.7, 0.1]
+})
+
+observable_df = pd.DataFrame(data={
+    OBSERVABLE_ID: ['obs_a'],
+    OBSERVABLE_FORMULA: ['A'],
+    NOISE_FORMULA: [0.5]
+}).set_index([OBSERVABLE_ID])
+
+parameter_df = pd.DataFrame(data={
+    PARAMETER_ID: ['k1', 'k2'],
+    PARAMETER_SCALE: [LIN] * 2,
+    LOWER_BOUND: [0] * 2,
+    UPPER_BOUND: [10] * 2,
+    NOMINAL_VALUE: [0.8, 0.6],
+    ESTIMATE: [1] * 2,
+}).set_index(PARAMETER_ID)
+
+
+# write files
+
+write_problem(test_id=test_id,
+              parameter_df=parameter_df,
+              condition_dfs=[condition_df],
+              observable_dfs=[observable_df],
+              measurement_dfs=[measurement_df])
+
+# solutions ------------------------------------------------------------------
+
+simulation_df = measurement_df.copy(deep=True).rename(
+    columns={MEASUREMENT: SIMULATION})
+simulation_df[SIMULATION] = [3 * analytical_a(t, 0, 1, 0.8, 0.6)
```

Comment:
Not completely sure what should happen if we have specified initial concentration in the model and increase the initial compartment size. I guess the concentration would stay constant? With specified initial amounts, it should be diluted.

Reply:
What do you mean here? Is the test fine, should we add another one, ..., or do we need to fine that behavior properly?